internal/provisioner/plain/main.go

```diff
@@ -146,6 +146,10 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "BundleInstance")
 		os.Exit(1)
 	}
+	if err = (&rukpakv1alpha1.Bundle{}).SetupWebhookWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create webhook", "webhook", "Bundle")
+		os.Exit(1)
+	}
```

Comment:
I think we unfortunately need a separate main.go and deployment for the core webhooks related to the APIs. This validation isn't a concern of the plain provisioner. Its a concern of the core APIs.

Reply:
I see. I'll try.  Can it be placed at internal/core?

Reply:
I added the new main.  I put it in the same image and created another deployment/pod for it.  Please review.  Thanks!